18-dualstack_ipv4_ipv6_seed_shoot.md

```diff
@@ -27,7 +27,23 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
 
 ### Configuration
 
-> TODO
+In the shoot specification we will use the definition of the `network` section to provide the cidrs for `nodes`, `pods` and `services`. As those fields already exist they will be enhanced to accept a tuple of values. One value for IPv4 and one for IPv6. We introduce special key words to request different strategies for choosing the respective CIDRs. These will be available at runtime only and being configured by Gardener/K8S during the cluster deployment/reconciliation.
+
+The following keywords are currently anticipated
+- **`autoV4`/`autoV6`** use the cloud provider's IPAM to reserve a prefix for the range (optional: of at least that size).
```

Comment:
autoV4 will always be private ranges while autoV6 will probably be a public range. This is quite a difference. when using IPv4 addresses will not be managed or assigned by the cloud provider